app/src/main/java/com/project/myproject/data/repository/ContactRepository.kt

```diff
@@ -0,0 +1,15 @@
+package com.project.myproject.data.repository
+
+import com.project.myproject.data.room.dao.ContactDao
+import com.project.myproject.data.room.entities.Contact
+import javax.inject.Inject
+
+class ContactRepository @Inject constructor(private val contactDao: ContactDao) {
```

Comment:
I suppose you've violated dependency inversion principle somewhere down in your code